spec/requests/editions_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+RSpec.describe "Editions" do
+  describe "POST /document/:document/editions" do
+    it "creates a new edition" do
+      edition = create(:edition, :published)
+      stub_publishing_api_put_content(edition.content_id, {})
+      post create_edition_path(edition.document)
+      follow_redirect!
+
+      expect(response.body).to include(I18n.t!("documents.edit.title", title: edition.title))
```

Comment:
This expectation doesn't capture that we've created a new edition - checking the title seems somewhat incidental. Might make more sense to have a 

```
expect { post create_edition_path(edition.document) }
  .to change { Edition.where(document_id: document.id).count }.by(1)
```

Reply:
Good catch!